integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/iceberg/SparkIcebergCatalogIT.java

```diff
@@ -188,12 +188,12 @@ void testIcebergPartitions() {
 
               String insertData =
                   String.format(
-                      "INSERT into %s values(2,'a',cast('2024-01-01 12:00:00.0' as timestamp));",
+                      "INSERT into %s values(2,'a',cast('2024-01-01 12:00:00.0' as timestamp_ntz));",
                       tableName);
               sql(insertData);
               List<String> queryResult = getTableData(tableName);
               Assertions.assertEquals(1, queryResult.size());
-              Assertions.assertEquals("2,a,2024-01-01 12:00:00.0", queryResult.get(0));
+              Assertions.assertEquals("2,a,2024-01-01T12:00", queryResult.get(0));
```

Comment:
Could we specify time zones only to fix this issue? cc @FANNG1

Reply:
for example, sql("SET spark.sql.session.timeZone=Asia/Shanghai");

Reply:
This is just a test case, we just need to make it return the correct and same value on all machines for `timestamp` type. WDYT?

Reply:
yes, it's  just a test case, let me think more

Reply:
using a global UTC timezone, mainly because `timestamp_ntz` is only available after Spark 3.3. thanks for advising.